### PR TITLE
feat: finalised login page ui

### PR DESCRIPTION
**Pull request by waynewee**
- finalised login page ui (change text, alignment and tests)

### CODE REVIEW

File: src/components/Home/WalletDecryption/WalletDecryption.tsx [hunk 1]
```diff
@@ -26,28 +26,25 @@ export const WalletDecryption: FunctionComponent<WalletDecryption> = ({
   };
   const isDecrypting = decryptProgress > 0 && decryptProgress < 1;
 
-  const inputBorderCSS = isIncorrectPassword
-    ? "w-full border-solid border border-red h-10 p-3"
-    : "w-full border-solid border border-gray-300 h-10 p-3";
+  let inputBorderCSS = `w-full border-solid border h-10 p-3 rounded-lg`;
+
+  inputBorderCSS += isIncorrectPassword ? ` border-red` : ` border-gray-300`;
```
Comment (isaackps) [inline src/components/Home/WalletDecryption/WalletDecryption.tsx:31]: can change the red and gray to the new colors?

File: src/components/Home/WalletDecryption/WalletDecryption.tsx [hunk 2]
```diff
@@ -26,28 +26,25 @@ export const WalletDecryption: FunctionComponent<WalletDecryption> = ({
...
   return (
     <Wrapper>
-      <Title className="mb-8">Create Document</Title>
+      <Title className="mb-8">Create and Revoke Document</Title>
       <ContentFrame>
         <Card>
           <form className="relative flex flex-col rounded">
             {isDecrypting && <BarTrack progress={decryptProgress} className="absolute top-0 left-0" />}
-            <div className="text-gray-800 mr-4 mb-4 font-bold text-lg" data-testid="login-title">
+            <div className="text-gray-800 mr-4 mb-4 text-2xl" data-testid="login-title">
```
Comment (isaackps) [inline src/components/Home/WalletDecryption/WalletDecryption.tsx:41]: like wise for all the colors in this file

File: src/components/Home/WalletDecryption/WalletDecryption.tsx (hunk 1 above)
Comment (waynewee) [reply src/components/Home/WalletDecryption/WalletDecryption.tsx:31]: red -> rose
gray-300 -> cloud-200?
double checking cos not specified on figma for this particular component

**Review by isaackps — APPROVED** [review]
lgtm!

**Pull request merged**